### ISSUE TITLE
3.1.1. Captains and Choosing a Co-co-captain

### ISSUE THREAD

**Issue by ARMediting** [gryphonshafer/Quizzing-Rule-Book#173]
Regardless of what gets decided over at issue #123 [note that its references to 2.1.1. are to the same section in the RB], I'd like to adjust something else in the second paragraph of 3.1.1.:

> Only the captain or co-captain may challenge a decision or respond to a challenge. If both the captain and co-captain have erred-out, another quizzer can be designated by the coach to respond to or initiate challenges.

It seems unfair that a team with two erred-out quizzers can appoint another captain, but nothing allows a team with one quizzed-out and one erred-out quizzer to appoint a captain.

I propose two options: First, change the language in 3.1.1. to allow for more scenarios. (This proposed language allows designation of a co-co-captain if the reason for removal is substitution without a quiz out or error out, but that could be added if we'd like.)

> [3.1.1.] Only the captain or co-captain may challenge a decision or respond to a challenge. If both the captain and co-captain have **_left the platform_**, another quizzer can be designated by the coach to respond to or initiate challenges.

Or, alternatively, change the language in 5.7. Substitutions to allow quizzed out quizzers to return to the platform. This could be done by only allowing them back in if there's an error out or foul out, or just allow any substitution.

> [5.7.] Quizzed-out quizzers may remain in the quiz to answer bonus questions. Substituting for them afterward will require a timeout. If subbed-out, they may not return for any reason.

If the quizzed-out quizzer can return to the bench for any reason, that could lead to possible substitutions for bonus questions, but I say why not allow for that strategy? Put most simply, a team shouldn't be penalized because of a quizzed-out captain, but looking a little wider, why not give a coach more freedom to utilize quizzer position, even when a quizzer is quizzed out?

**Comment by jttower** [comment]
Can anyone explain the second part of 5.7 that says they may not return for any reason?  Would the next sentence supersede this rule?
_When a quizzer quizzes-out, errs-out, or fouls-out and leaves the platform after the event, a substitute may take their place immediately and without a timeout, even if the incoming quizzer has been subbed-out less than 3 questions ago._
I guess I'm confused about them not being allowed to return for any reason if they are subbed out, because if someone subsequently errs out, then doesn't this sentence mean the quizzed out quizzer can return to the bench immediately and without a timeout?

**Comment by kclimenhaga** [comment]
As it stands (at least in my understanding), if one quizzer quizzes out and leaves the platform, and then a second quizzer errs out, the quizzed-out quizzer is not allowed to return to take the empty place. 

I don't like the idea of a quizzed out quizzer being allowed back on as a general rule, because I think that all quizzers should have as much time as possible on the platform, and I don't want to see a quizzed out quizzer replacing a quizzer who is still able to jump. However, I am not opposed to a quizzed out quizzer being allowed to replace an erred out quizzer.

I am also not opposed to a quizzed-out captain being allowed to challenge (though my first inclination is that an erred-out captain should not be allowed to challenge). I assume the reason the quizzed-out captains aren't allowed to challenge is because we don't want a coach to be able to prompt the quizzer to challenge, but I don't think that is a major concern.

That being said, I like Alex's first suggestion of just changing the wording on co-captains, since that is simpler and will cause fewer complications.

**Comment by jttower** [comment]
I still don't understand why a quizzed-out quizzer can't return to the platform IF a quizzer errs out.  I would prefer to clarify things to allow them to return in this instance.  Though as I stated before, I have always assumed 5.7 means they can return in this instance, so I guess we need to clarify that.

**Comment by ARMediting** [comment]
@jttower I agree that I'd like to see that happen. I didn't understand the rules as currently allowing that, and I was just asking whether people would prefer to only allow quizzed-out quizzers back in after an error out, or open wide allowances for coaches to move quizzed out quizzers as they'd like. (I am against allowing erred-out quizzers back to the quiz and against a quizzer challenging from the audience.)

**Comment by levikoral** [comment]
Co-captains should not be allowed to challenge as there are many "bad" challenges that occur if they should choose to challenge.
